components/authz-service/server/v2/project.go

```diff
@@ -595,13 +595,13 @@ func fromAPIType(t api.ProjectRuleTypes) (storage.RuleType, error) {
 func rulesToProjectRules(rules []engine.Rule) ([]*api.ProjectRule, error) {
 	conditions := make([]*api.Condition, len(rules))
 	for j, rule := range rules {
-		conditionType, exists := api.ProjectRuleConditionTypes_value[rule.Type]
+		conditionAttribute, exists := api.ProjectRuleConditionAttributes_value[rule.Type]
```

Comment:
The thing used to index into `api.ProjectRuleConditionAttributes_value` is an attribute, not a type, now, so not sure why this should work as written.

Reply:
This is getting deleted in https://github.com/chef/automate/pull/517 so I didn't go beyond fixing the compiler error